### PR TITLE
Feat/inline video

### PR DESCRIPTION
**Pull request by nehasri89**
This PR adds inline videos

### CODE REVIEW

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/ad0be18b690dc30132fdd19d96a59e32c4a506a5/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/53d39117867d5244982f20cd9e15b7062b9e5055/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/65dfdab9916531a98e8cea21105cb2bfa52de4ee/index.html

File: packages/article/src/article-body/media-aspect-ratio.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { View } from "react-native";
+import PropTypes from "prop-types";
+
+const AspectRatioContainer = ({ aspectRatio, children }) => {
```
Comment (BernieSumption) [inline packages/article/src/article-body/media-aspect-ratio.js]: 👍

File: packages/article/src/article-body/article-body-row.js [hunk 2]
```diff
@@ -1,11 +1,21 @@
...
 import ArticleImage from "@times-components/article-image";
 import PullQuote from "@times-components/pull-quote";
+import AspectRatioContainer from "./media-aspect-ratio";
 import BodyParagraph from "./article-body-paragraph";
 import ArticleLink from "./article-link";
+import InsetCaption from "./inset-caption";
+
+const primaryContainer = {
+  width: "100%",
+  flexDirection: "column",
```
Comment (craigbilner) [inline packages/article/src/article-body/article-body-row.js]: this is the default?

File: packages/article/src/article-body/article-body-row.js [hunk 3]
```diff
@@ -56,6 +66,32 @@ const ArticleRow = ({ content: { data, index } }) =>
           {children}
         </ArticleLink>
       );
+    },
+    video(
```
Comment (craigbilner) [inline packages/article/src/article-body/article-body-row.js]: perhaps not for this PR but would it be worthwhile to do:

```
const render1 = () =>
const render2 = () =>
const render3 = () =>

renderTrees([data], {
  render1,
  render2,
  render3,
});

Perhaps easier to read and refactor?

File: packages/article/src/article-body/article-body-row.web.js [hunk 4]
```diff
@@ -14,8 +16,8 @@ import {
   PullQuoteResp
 } from "../styles/article-body/responsive";
 
-export const responsiveImageWrapper = imageType => {
-  switch (imageType) {
+export const responsiveDisplayWrapper = displayType => {
```
Comment (craigbilner) [inline packages/article/src/article-body/article-body-row.web.js]: could we address the code duplication separately?

File: packages/article/src/article-body/media-aspect-ratio.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+  const [ratioWidth, ratioHeight] = aspectRatio.split(":");
```
Comment (craigbilner) [inline packages/article/src/article-body/media-aspect-ratio.js]: utils?

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/db292aa8a31a6ab0324bb3e34593f3ef4457211b/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/4d2366fcce1cf26775e96eef708c72bf0fe9c5e3/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/59d256db4ac7f1192784c47265dc9808bdb64147/index.html

**Pull request merged**